components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -83,7 +87,10 @@
                       </div>
                       <div v-if="nftRoyalties" class="royalty">
                         ⊆ {{ $t('royalty') }}
-                        <Money :value="nftRoyalties" inline />
+                        <Money
+                          :value="nftRoyalties"
+                          inline
+                          :prefix="urlPrefix" />
```

Comment:
```suggestion
       />
```
`prefix` has not been used